lib/board.rb

```diff
@@ -0,0 +1,39 @@
+# rubocop: disable Metrics/AbcSize
+
+# rubocop: disable Metrics/CyclomaticComplexity
+
+# rubocop: disable Metrics/PerceivedComplexity
+
+class Board
+  attr_accessor :board
```

Comment:
- [ ] You shouldn't use  `attr_accessors` unless you really need to. Using `attr_accessors` means that the property can be written to/changed from anywhere including from outside the class.
screenshot of requirements 👇 
![Screen Shot 2020-08-28 at 1 12 56 AM](https://user-images.githubusercontent.com/48804258/91503751-9c821f80-e8cb-11ea-8c8c-8df1d3348157.png)

Reply:
:100: